src/panel.js

```diff
@@ -291,22 +360,39 @@ const buildPanel = async(panelId) => {
     addLearnMoreLink(contentWrapper);
   }
 
-  addLearnHowFBCWorksButton(fragment);
+  await addLearnHowFBCWorksButton(fragment);
+
+  if (panelId === "no-trackers") {
+    await addCustomSiteButton(fragment, panelId);
+  }
```

Comment:
This is called below on line 369 whether or not `panelId === "no-trackers"` - so do we need this if block?

Reply:
Yes – It's a quirk for how its' rendered in the panel. I tried different combinations/ordering of these.